core/core-funcs.el

```diff
@@ -253,5 +253,11 @@ result, incrementing passed-tests and total-tests."
      (concat "Hidden Mode Line Mode enabled.  "
              "Use M-x hidden-mode-line-mode to make the mode-line appear."))))
 
+(defun spacemacs/recompile-elpa ()
+  "Recompile packages in elpa directory. Useful if you switch
+Emacs versions."
+  (interactive)
+  (byte-recompile-directory package-user-dir nil t))
```

Comment:
I always saw `'append` or `'force` used, maybe `:force` is a clever choice but it looks weird to me since keyword are used for keyword arguments, it feels like the opposite of what we want, i.e. the `:force` argument is nil(since there is nothing after it).

The `'append` trick is used for boolean like arguments so it should not be an issue to pass a sexp instead of a constant, if the function evaluates what is expected to be a constant, it looks to me that the function argument choice (or docstring) is wrong in the first place.